recipes/freeimage/all/conanfile.py

```diff
@@ -78,12 +78,13 @@ def requirements(self):
         elif self.options.with_jpeg == "libjpeg-turbo":
             self.requires("libjpeg-turbo/2.1.2")
         if self.options.with_jpeg2000:
-            self.requires("openjpeg/2.4.0")
+            self.requires("openjpeg/2.5.0")
         if self.options.with_png:
             self.requires("libpng/1.6.37")
         if self.options.with_webp:
             self.requires("libwebp/1.2.2")
         if self.options.with_openexr:
```

Comment:
```suggestion
        if self.options.with_tiff or self.options.with_openexr:
```

Reply:
I don't like options like `with_<dependency>`, because it says nothing of feature you get (inheritance of lack of package manager in C/C++ I guess, where everybody is afraid by dependencies, so it's more important to know what kind of dependency is added than the enabled feature). So here if we understand `with_<something>` as `with_plugin_<something>`, it makes sense to add dependencies required for a plugin.

Reply:
I totally agree with you.
I will try to improve this PR.